packages/razorsharp/src/code/blade/constants/icons.ts

```diff
@@ -0,0 +1,272 @@
+export const BLADE_ICONS = [
```

Comment:
It might get difficult to maintain the parity of icon names in this module, you should perhaps figure out a way to pick these values from blade package as the source of truth.

Reply:
Yes. agreed. This would be a part of our discussion next thursday in office hours